common/interpolation/include/interpolation/interpolation_utils.hpp

```diff
@@ -15,48 +15,78 @@
 #ifndef INTERPOLATION__INTERPOLATION_UTILS_HPP_
 #define INTERPOLATION__INTERPOLATION_UTILS_HPP_
 
+#include <algorithm>
 #include <array>
+#include <functional>
+#include <limits>
 #include <stdexcept>
 #include <vector>
 
 namespace interpolation_utils
 {
+constexpr double EPS = std::numeric_limits<double>::epsilon();
```

Comment:
IMO, this global epsilon should be removed.
How small the epsilon should be depends on each algorithm.
Therefore, it will be confusing if there is a global epsilon variable.

Reply:
Thank you very much, we can remove it, after your commits I remove it.

Reply:
I finished my change. Please modify!

Reply:
I couldn’t have done it without you. I have removed EPS.